create_aio_app/template/{{cookiecutter.project_name}}/{{cookiecutter.project_name}}/routes.py

```diff
@@ -12,7 +12,6 @@ def init_routes(app: web.Application) -> None:
 
     add_route('*', '/', index, name='index')
 
-    # added static dir
```

Comment:
Why did u remove this line?

Reply:
What is the purpose of this comment? It is pretty clear from code that we are adding static dir. But i can revert no prob.

Reply:
Please return if you can

Reply:
done

Reply:
but i still doubt about this comment :)

Reply:
@hzlmn removing a bunch of unrelated changes is a smelly practice. Avoid doing this.

Reply:
@webknjaz Thanks, i will take note.